### PR TITLE
Feature/data photos

### PR DESCRIPTION
**Pull request by Elgin-Schultz**
Only adding data and Photos.

### CODE REVIEW

File: components/GlobalStyles.js [hunk 1]
```diff
@@ -55,27 +56,27 @@ a {
 input, button, submit
```
Comment (julialoeschel) [inline components/GlobalStyles.js]: Do you use any inputs or submits in your app?

File: components/GlobalStyles.js [hunk 2]
```diff
@@ -55,27 +56,27 @@ a {
...
-.document{
-   padding:2%;
-   background-color: #D0B3B3;  
-}
+
+.info-card {
+    background-color: white;
+    padding: 1rem;
+}  
+.info {font-family: 'Trade Gothic';
```
Comment (julialoeschel) [inline components/GlobalStyles.js:79]: is prettier not working here?

File: pages/singlesite/[slug].js [hunk 3]
```diff
@@ -34,17 +34,36 @@ export default function singleSite() {
...
+            direction={"vertical"}
+            spaceBetween={0}
+            slidesPerView={1}
+            pagination={{
+              clickable: true,
+            }}
+            modules={[Pagination]}
+          >
+            <StyledSwiperSlide>
+              <div className="info-card">
```
Comment (julialoeschel) [inline pages/singlesite/[slug].js:50]: please do not mix classNames and styled componenst. Only if you need to

File: pages/singlesite/[slug].js [hunk 4]
```diff
@@ -34,17 +34,36 @@ export default function singleSite() {
...
+                  {currentSite.coordinates[0] +
+                    " | " +
+                    currentSite.coordinates[1]}{" "}
+                </h2>
+                <h2>{currentSite.adress}</h2>
+                <p className="info">{currentSite.information[0]}</p>
+              </div>
+            </StyledSwiperSlide>
+
+            <StyledSwiperSlide>
```
Comment (julialoeschel) [inline pages/singlesite/[slug].js:61]: do you always have 2 information entires?  I suggest to add some logic to make this dynamic ;)

File: pages/singlesite/[slug].js [hunk 5]
```diff
@@ -210,8 +236,8 @@ const StyledSwiperSlide = styled(SwiperSlide)`
 const StyledImage = styled(Image)`
   display: block;
   object-fit: contain;
-  width: 80% !important;
-  height: 80% !important;
+  width: 100% !important;
```
Comment (julialoeschel) [inline pages/singlesite/[slug].js]: it seems that everything is very important ;) we usually try to avoid using `!important`.

**Review by julialoeschel — COMMENTED** [review]
Hey There, good job!

we have some suggestions for you .

Also, as you mentioned earlier today, we found lots of static code (repeating code) (which was not part of this PR. )

Try to generate your code more dynamically to render pictures for example. Let us know if you need further support on this :) 

I like the slider, its really cool!
🐕

File: pages/singlesite/[slug].js (hunk 5 above)
Comment (Elgin-Schultz) [reply pages/singlesite/[slug].js]: I am checking that !

File: pages/singlesite/[slug].js (hunk 3 above)
Comment (Elgin-Schultz) [reply pages/singlesite/[slug].js:50]: SwiperJs is asking for ClassNames here, but will check when I do code refactoring!

File: components/GlobalStyles.js (hunk 2 above)
Comment (Elgin-Schultz) [reply components/GlobalStyles.js:79]: Nooooooo

File: components/GlobalStyles.js (hunk 1 above)
Comment (Elgin-Schultz) [reply components/GlobalStyles.js]: Deleated!

**Pull request merged**